### PR TITLE
chore: CKP IFO banner

### PR DESCRIPTION
**Pull request by Chef-Yogi**
---

### Detailed summary
- Added a new translation for "CKP cIFO" in the `translations.json` file.
- Updated the image URLs for `liquidStakingBunnyBg3` and `liquidStakingBunnyBg4` in `WebNotificationBanner.tsx`.
- Reordered the imports and updated the import statement for `IFOBanner` to `NewIFOBanner` in `useMultipleBannerConfig.tsx`.
- Updated the `useCountdown` hook in `useCountdown.ts` by reordering the import statements.
- Added a new component `NewIFOBanner.tsx` with multiple styled components and images, including a countdown component.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx [hunk 1]
```diff
@@ -41,10 +41,11 @@ export const useMultipleBannerConfig = () => {
...
       },
-      {
-        shouldRender: isRenderIFOBanner,
-        banner: <IFOBanner />,
-      },
+      // {
+      //   shouldRender: isRenderIFOBanner,
+      //   banner: <IFOBanner />,
+      // },
+      { shouldRender: true, banner: <NewIFOBanner /> },
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx:48]: No have a hook to check when should we show the banner?

File: apps/web/src/views/Home/components/Banners/NewIFOBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+            justifyContent="center"
+            alignItems="center"
+            style={{ gap: isMobile ? 4 : 16 }}
+          >
+            <Image src={partnerCakePie} alt="pancakeswap-logo" width={121} height={25} />
+            <Divider />
+            <Image src={cakeLogo} alt="liquidStakingTitle" width={20} height={20} />
+            <Image src={partnerBnbChain} alt="liquidStakingTitle" width={104} height={24} />
+          </Flex>
+          <StyledSubheading data-text={t('Web3 Notifications (BETA) trial available')}>
```
Comment (chefjackson) [inline apps/web/src/views/Home/components/Banners/NewIFOBanner.tsx:213]: Shall we remove?

File: apps/web/src/views/Home/components/Banners/NewIFOBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+            {t('CKP cIFO')}
+          </StyledSubheading>
+          <Countdown />
+          <Flex style={{ gap: isMobile ? 4 : 16 }}>
+            <Link
+              href="https://blog.pancakeswap.finance/articles/introducing-web3-notifications-on-pancake-swap"
```
Comment (chefjackson) [inline apps/web/src/views/Home/components/Banners/NewIFOBanner.tsx:219]: incorrect link

File: apps/web/src/views/Home/components/Banners/NewIFOBanner.tsx [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+  height: 296px;
+  border-radius: 296px;
+  background: radial-gradient(50% 50% at 50% 50%, #beb1ce 0%, rgba(190, 177, 206, 0) 100%);
+  filter: blur(64.5px);
+`
+
+export function Countdown() {
+  const { t } = useTranslation()
+  const { isMobile } = useMatchBreakpoints()
+  const countdown = useCountdown(1704376800)
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/NewIFOBanner.tsx:173]: Wrong time. Should be this check IFO config.
`1704369600, // Thu Jan 04 2024 12:00:00 UTC`

File: packages/hooks/src/useCountdown.ts [hunk 5]
```diff
@@ -46,3 +46,10 @@ export function useCountdown(to: number): Countdown | undefined {
 
   return countdown
 }
+
+export const useIsCountdownOver = (to: number) => {
+  const target = dayjs.unix(to)
+  const relative = dayjs()
+  const diff = useMemo(() => target.diff(relative, 'second'), [target, relative])
```
Comment (chefjackson) [inline packages/hooks/src/useCountdown.ts:53]: `relative` is always a new instance once this hook is called so the `useMemo` here is not doing its job.

Btw, I think we can reuse the `useCountdown` hook?

Comment (chefjackson) [reply packages/hooks/src/useCountdown.ts:53]: just need to check if all the numbers are 0

Comment (Chef-Yogi) [reply packages/hooks/src/useCountdown.ts:53]: Sure, I will merge it to useCountdown

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**